### PR TITLE
Update 4.8.0 framework changelog

### PR DESCRIPTION
**Pull request by Selutario**
|Related issue|
|---|
|#20077|

## Description
Update framework changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,23 +15,53 @@ All notable changes to this project will be documented in this file.
 - Added timeout and retry parameters to the VirusTotal integration. ([#16893](https://github.com/wazuh/wazuh/pull/16893))
 - Extended wazuh-analysisd EPS metrics with events dropped by overload and remaining credits in the previous cycle. ([#18988](https://github.com/wazuh/wazuh/pull/18988))
 - Replaced Filebeat's date index name processor. ([#19819](https://github.com/wazuh/wazuh/pull/19819))
+- Updated API and framework packages installation commands to use pip instead of direct invocation of setuptools. ([#18466](https://github.com/wazuh/wazuh/pull/18466))
+- Upgrade docker-compose V1 to V2 in API Integration test scripts. ([#17750](https://github.com/wazuh/wazuh/pull/17750))
```
Comment (GGP1) [inline CHANGELOG.md:19]:
```suggestion
- Upgraded docker-compose V1 to V2 in API Integration test scripts. ([#17750](https://github.com/wazuh/wazuh/pull/17750)) 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,23 +15,53 @@ All notable changes to this project will be documented in this file.
 - Added timeout and retry parameters to the VirusTotal integration. ([#16893](https://github.com/wazuh/wazuh/pull/16893))
 - Extended wazuh-analysisd EPS metrics with events dropped by overload and remaining credits in the previous cycle. ([#18988](https://github.com/wazuh/wazuh/pull/18988))
 - Replaced Filebeat's date index name processor. ([#19819](https://github.com/wazuh/wazuh/pull/19819))
+- Updated API and framework packages installation commands to use pip instead of direct invocation of setuptools. ([#18466](https://github.com/wazuh/wazuh/pull/18466))
+- Upgrade docker-compose V1 to V2 in API Integration test scripts. ([#17750](https://github.com/wazuh/wazuh/pull/17750)) 
+- Refactored how cluster status dates are treated in the cluster. ([#17015](https://github.com/wazuh/wazuh/pull/17015))
+
+#### Fixed
+- Delete cluster temporary files when the connection between a worker and a master is broken. ([#17886](https://github.com/wazuh/wazuh/pull/17886))
```
Comment (GGP1) [inline CHANGELOG.md:23]:
```suggestion
- Forced the deletion of cluster temporary files when the connection between a worker and a master is broken. ([#17886](https://github.com/wazuh/wazuh/pull/17886))
```

or

```suggestion
- Updated cluster connection cleanup to remove temporary files when the connection between a worker and a master is broken. ([#17886](https://github.com/wazuh/wazuh/pull/17886))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,23 +15,53 @@ All notable changes to this project will be documented in this file.
...
+
+### Other
+
+#### Changed
+
+- Upgraded external aiohttp library dependency version to 3.8.5. ([#20003](https://github.com/wazuh/wazuh/pull/20003))
+- Upgraded external cryptography library dependency version to 41.0.4. ([#20003](https://github.com/wazuh/wazuh/pull/20003))
+- Upgraded external numpy library dependency version to 1.26.0. ([#20003](https://github.com/wazuh/wazuh/pull/20003))
+- Upgraded external grpcio library dependency version to 1.58.0. ([#20003](https://github.com/wazuh/wazuh/pull/20003))
+- Upgraded embedded Python version to 3.10.13. ([#20003](https://github.com/wazuh/wazuh/pull/20003))
```
Comment (GGP1) [inline CHANGELOG.md:62]: There were more dependencies upgrades, but I'm assuming we just include the more relevant ones or the ones that fixed a vulnerability?

**Review by GGP1 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (Selutario) [reply CHANGELOG.md:62]: That's right, I only included some of them, especially those that fixed vulnerabilities that are not already being fixed in 4.7.1.

**Review by GGP1 — APPROVED** [review]

**Pull request merged**